blaze/interactive.py

```diff
@@ -41,19 +41,27 @@
     not_an_iterator.append(pymongo.database.Database)
 
 
-class InteractiveSymbol(Symbol):
-    """Interactive data.
+class _Data(Symbol):
```

Comment:
should we just call this `Data` to match the other expression/smart constructor pairs?